core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -4539,9 +4584,9 @@ private void processAsyncWriterCommand(
                     .I$();
             errorCode = STRUCTURE_CHANGE_NOT_ALLOWED;
             errorMsg = "async cmd cannot change table structure while writer is busy";
-        } catch (SqlException | CairoException ex) {
-            errorCode = SQL_OR_CAIRO_ERROR;
-            errorMsg = ex.getFlyweightMessage();
+        } catch (CairoException ex) {
+            errorCode = CAIRO_ERROR;
+            errorMsg = "async cmd failed: " + ex.getFlyweightMessage();
```

Comment:
"async cmd failed: " is redundant, this message can be added on the receiving end

Reply:
Removed in a19fb021d74ac8c4dc4b42ced8564aee432b3c8d